src/python/tests/conftest.py

```diff
@@ -32,5 +32,8 @@ def pytest_configure() -> None:
 
     """
     test_data_dir = Path(__file__).parent
+    root_dir = test_data_dir / "../../.."
```

Comment:
```suggestion
    root_dir = test_data_dir.parents[2]
```

Reply:
Shouldn't it be test_data_dir.parents[3]?

Reply:
`test.parents[0]` already returns you the first parent (based on my understanding from the documentation), so "0-based index" here as well.

Reply:
OK, now done